tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -723,6 +723,10 @@ _ when x.ToBoolean() is { } boolean => boolean,
                 telemetry.Record(ConfigTelemetryData.AasAppType, AzureAppServiceMetadata.SiteType, recordValue: true, ConfigurationOrigins.Default);
             }
 
+            GraphQLErrorExtensions = TrimSplitString(
```

Comment:
Non-blocking, but it would be nice to add a unit test to assert expected inputs/outputs. Also, I expect that we _should_ be deduplicating entries (e.g `DD_TRACE_GRAPHQL_ERROR_EXTENSIONS=code,code`) but this doesn't appear to do any deduplication